src/server/worldserver/worldserver.conf.dist

```diff
@@ -2516,6 +2519,14 @@ Rate.MoveSpeed = 1
 
 Rate.InstanceResetTime = 1
 
+#
+#    Rate.Pet.LevelXP
+#        Description: Modifies the amount of experience required to leveling up a pet.
```

Comment:
```suggestion
#        Description: Modifies the amount of experience required to level up a pet.
```

Reply:
I think think it's the experience received by the player that is shared to the pet right? then need to improve description if so and mention the player/character's exp